src/tests/gfx_helpers.edn

```diff
@@ -0,0 +1,76 @@
+(def timestep (/ 1.0 120.0))
```

Comment:
Might want to also rename the file `gfx-helpers.edn` to follow the naming of the other tests script.

Same with `gfx-highlight.edn`.